_docs/data-model/v1/upload.md

```diff
@@ -55,6 +59,48 @@ At some point, we may support real-time monitoring of devices, in which case the
 
 The Tidepool platform supports the idea that a user other than the data owner may have upload permissions (for example, a clinic or a parent). The byUser field should be the actual userId of the user doing the upload. This field will be stored in encrypted form in the actual data stream as it constitutes PII.
 
+## device information
+
+### deviceType
+This is the broad category of the device. In general, it should be one of the following:
+
+* `"pump"` (for insulin pumps)
+* `"cgm"` (for continuous glucose monitors)
+* `"pump-cgm"` (for pumps integrated with a cgm)
+* `"bgm"` (for blood glucose meters that are not continuous meters)
+* `"pen"` (for insulin injection devices)
+* `"manual"` (for manually entered data)
+
+For devices that do not fit in one of these categories, please contact Tidepool so we can standardize on a good name.
+
+### deviceManufacturer
+
+This is the name of the device manufacturer. We are trying to standardize this field as well, so please use exactly the following names:
+
+    Abbott
+    Asante
+    Dexcom
+    Insulet
+    Medtronic
+    OneTouch
+
+Again, for new devices please contact Tidepool so we can standardize on a name.
+
+### deviceSeries and deviceModel
+
+Manufacturers often identify devices in a series of similar devices, each with a different model identification -- for example, Medtronic has the Paradigm Revel 523 and Paradigm Revel 723 pumps. Therefore, we have two levels of identifier. In this case, `deviceSeries` would be `"Paradigm Revel"` and `deviceModel` would be `523` or `723`.
+
+For manufacturers who don't have multiple levels, use the deviceModel field and set the deviceSeries field to an empty string (do not omit it).
```

Comment:
What's your reasoning here? I think it might be easier to deal with the data if all the info is generally available in the `deviceSeries` field for all devices, regardless of whether they have the two levels. The second level of info is necessary to be precise but always irrelevant in terms of the kind of functionality we care about it.